vispy/gloo/texture.py

```diff
@@ -48,19 +48,37 @@ class BaseTexture(GLObject):
     _inv_formats = {
         'luminance': 1,
         'alpha': 1,
+        'red': 1,
         'luminance_alpha': 2,
+        'rg': 2,
         'rgb': 3,
-        'rgba': 4,
+        'rgba': 4
```

Comment:
I have a slight preference for leaving the extra `,` in place so that if something is appended, you only see the added line instead of an added and a changed line, but I can live with the change :)